### PR TITLE
Add test to make sure reviews are deleted when the shelter they belon…

### PR DESCRIPTION
**Pull request by agallant121**
…g to are also deleted.

### CODE REVIEW

File: spec/features/shelters/cannot_delete_with_approved_pets_spec.rb [hunk 1]
```diff
@@ -9,6 +9,9 @@
     @shelter_1 = Shelter.create!(name: "Abby's Shelter", address: "123 Maine Street", city: "Denver", state: "CO", zip: "80210")
     @shelter_2 = Shelter.create!(name: "Alex's Shelter", address: "123 Maine Street", city: "Bangor", state: "ME", zip: "04401")
 
+    @review_1 = @shelter_2.reviews.create(title: "Bad shelter", rating: 1, content: "It had fleas")
+    @review_2 = @shelter_1.reviews.create(title: "Great shelter", rating: 1, content: "It had fleas")
```
Comment (agallant121) [inline spec/features/shelters/cannot_delete_with_approved_pets_spec.rb]: Added a second review belonging to a shelter that I was not deleting, so I could test the total number of reviews to make sure that the review was deleted when the shelter

File: spec/features/shelters/cannot_delete_with_approved_pets_spec.rb [hunk 2]
```diff
@@ -68,4 +71,21 @@
 
       expect(page).to_not have_content(@pet_2.name)
   end
+
+
+  it "deletes the reviews for the shelter when the shelter is deleted" do
+    reviews_count = Review.count
```
Comment (agallant121) [inline spec/features/shelters/cannot_delete_with_approved_pets_spec.rb]: Make sure that both reviews in the setup were created.

File: spec/features/shelters/cannot_delete_with_approved_pets_spec.rb [hunk 3]
```diff
@@ -68,4 +71,21 @@
...
+    expect(reviews_count).to eq(2)
+
+    visit "/shelters"
+      within("#shelter-#{@shelter_2.id}") do
+        click_link "Delete #{@shelter_2.name}"
+      end
+
+    reviews_count = Review.count
```
Comment (agallant121) [inline spec/features/shelters/cannot_delete_with_approved_pets_spec.rb]: Make sure the total reviews decreased when shelter was deleted, ensuring that that review was also deleted.

File: spec/features/shelters/cannot_delete_with_approved_pets_spec.rb [hunk 4]
```diff
@@ -68,4 +71,21 @@
...
+    expect(reviews_count).to eq(1)
+
+    review = Review.first
```
Comment (agallant121) [inline spec/features/shelters/cannot_delete_with_approved_pets_spec.rb]: Create new variable to call on the only review and make sure that the review does not have the content contained in the deleted review.

File: spec/features/shelters/cannot_delete_with_approved_pets_spec.rb (hunk 2 above)
Comment (adumortier) [reply spec/features/shelters/cannot_delete_with_approved_pets_spec.rb]: I like how you managed to test that

**Pull request merged**